<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>res</class>
 <widget class="QWidget" name="res">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1011</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>54</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>400</width>
     <height>320</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="pas">
      <property name="text">
       <string>密  码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namein"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pasin"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="name">
      <property name="text">
       <string>姓  名</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="phonein"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="pas2in"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="addr">
      <property name="text">
       <string>邮  箱</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="pow">
      <property name="text">
       <string>角  色</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="powin">
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>农民</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>公务员</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机器学习工程师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>农业专家</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>超级管理员</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="phone">
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="pas2">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="addrin"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="area">
      <property name="text">
       <string>地  区</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="pro">
         <item>
          <property name="text">
           <string>...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>湖南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>山西</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>陕西</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="city">
         <item>
          <property name="text">
           <string>...</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="submit">
      <property name="text">
       <string>注        册</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>251</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nameerr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="paserr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pas2err">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="addrerr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="phoneerr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="areaerr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="powerr">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
